molo/surveys/models.py

```diff
@@ -418,7 +418,11 @@ def has_page_breaks(self):
     def serve(self, request, *args, **kwargs):
         if (not self.allow_multiple_submissions_per_user and
                 self.has_user_submitted_survey(request, self.id)):
-            return render(request, self.template, self.get_context(request))
+            if request.method == 'POST' and 'ajax' in request.POST:
```

Comment:
```
if request.method == 'POST' and 'ajax' in request.POST:
     pass
else:
     return render(
             request, self.template, self.get_context(request))
```
change to `if not (request.method == 'POST' and 'ajax' in request.POST)`

Reply:
this will fail if it is a get request though?

Reply:
Yes